### PR TITLE
DeclareEqualNormalizeFixer - fix for declare having multiple directives

### PR DESCRIPTION
**Pull request by kubawerlos**
One more edgy edge case found running [regression](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6152#issue-1073467344).

### CODE REVIEW

**Review by julienfalque — APPROVED** [review]

File: tests/Fixer/LanguageConstruct/DeclareEqualNormalizeFixerTest.php [hunk 1]
```diff
@@ -87,6 +87,11 @@ public function provideFixCases(): array
                 null,
                 ['space' => 'none'],
             ],
+            'declare having multiple directives' => [
```
Comment (keradus) [inline tests/Fixer/LanguageConstruct/DeclareEqualNormalizeFixerTest.php:90]: let's have this case for both, `space=none` and `space=single`

Comment (kubawerlos) [reply tests/Fixer/LanguageConstruct/DeclareEqualNormalizeFixerTest.php]: done

**Review by SpacePossum — APPROVED** [review]

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.
